Formula/pnpm.rb

```diff
@@ -23,12 +23,35 @@ class Pnpm < Formula
   depends_on "node"
 
   def install
+    npmrc = OS.mac? ? "global-bin-dir = ${HOME}/Library/pnpm\n" : "global-bin-dir = ${HOME}/.local/pnpm\n"
+    (prefix/"etc/npmrc").write npmrc
```

Comment:
We can probably get around a conflicting `npmrc` if we do this in `post_install`. We'd need to:
1. Check if an `etc/"npmrc"` exists. If it doesn't exist, we can write this.
2. If it does exist, we need to check if `global-bin-dir` is set. If it isn't, we can append this line to the existing `npmrc`.

This should hopefully a) avoid installing a conflicting `npmrc`, and b) allow the user to use a different `pnpm` directory in case they don't want to use `$HOME/Library/pnmp` or `$HOME/.local/pnpm`.

Reply:
Of course, it would be much better if upstream just supported `pnpmrc`, but even then we want to account for users who want to use alternative directories instead of forcing the choice (or even adopting their own `etc/"pnpmrc"`).